src/USER-BOCS/testData/user-bocs.cmake

```diff
@@ -0,0 +1,8 @@
+# preset that turns on just a few, frequently used packages
```

Comment:
Please remove this file, too.

Reply:
Absolutely. Should never have made it into the repo.

Reply:
Folder and file both removed.